### ISSUE TITLE
Why does the car need to move forward after spinning left when it detects an obstacle

### ISSUE THREAD

**Issue by zhouxuan009** [zhouxuan009/CS179J-Team07#9]
I can imagine a possibility where the car doesn't move forward after spinning left when it detects an obstacle and the robot car get stuck into a dead loop. The car detects a obstacle and spin 45. Then it detects no object and turn its head to the destination but the angle should be -45. Then the car will get stuck into a dilemma.
Should we add a forward function after the car turn its head to the destination ?
How to set the moving forward speed in such two circumstums in order not to hit the obstacle?

**Comment by zhouxuan009** [comment]
Just set the refreshing time as 0.001s. I think it's ok. If the car spin 0.001s and then go forward 0.001s, there is no time for it to adjust itself. I want to modify the code. if the car has moved forward for like 0.3s, it can redict itself

**Comment by ctsan005** [comment]
You are right, this will give it higher response time, and the car will less likely get into a dead loop
